tests/phpunit/tests/test-order-methods.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * @covers \WC_Razorpay
+ */
+
+require_once __DIR__ . '/../mockfactory/MockApi.php';
+require_once __DIR__ . '/../mockfactory/Request.php';
+
+use Razorpay\MockApi\MockApi;
+
+class Test_OrderMethods extends WP_UnitTestCase
+{
+    private $instance;
+    private $rzpPaymentObj;
+    
+    public function setup(): void
+    {
+        parent::setup();
+
+        $this->rzpPaymentObj = new WC_Razorpay();
+
+        $this->instance = Mockery::mock('WC_Razorpay')->makePartial()->shouldAllowMockingProtectedMethods();
+
+        $_POST = array();
+    }
+
+    public function testgetRazorpayPaymentParams()
+    {
+        $order = wc_create_order();
+       
+        $wcOrderId = $order->get_id();
+
+        $this->instance->shouldReceive('getRazorpayApiInstance')->andReturnUsing(
+            function () {
+                return new MockApi('key_id_2', 'key_secret2');
+            });
+
+        $this->instance->shouldReceive('autoEnableWebhook');
+        
+        $razorpayOrderId=$this->instance->shouldReceive('createOrGetRazorpayOrderId')->with($wcOrderId)->andReturn('order_test');
+        
+        $this->assertEquals(['order_id'  =>  'order_test'] , $this->instance->getRazorpayPaymentParams($wcOrderId));
```

Comment:
@garimaprachi set one white space before and after =>, and no space before comma